src/components/v5/shared/UserPopover/partials/UserInfo.tsx

```diff
@@ -27,7 +28,12 @@ const UserInfo: FC<UserInfoProps> = ({
   const isTopContributorType = userStatus === 'top' && isContributorsList;
 
   return (
-    <div className="sm:min-w-[17rem]">
+    <div
+      className={clsx({
+        'min-w-[17rem]': !isTopContributorType,
+        'min-w-[20rem]': isTopContributorType,
```

Comment:
How does it look on mobile?

Reply:
Yeah, but the question is if it fits all the device sizes. On mobile it shouldn't be wider than screen size

Reply:
fixed